content/essentials/memos.mdx

```diff
@@ -102,7 +107,20 @@ createEffect(() => {
 });
 ```
 
-In the revised approach, the `isEven` memo will purely compute based on whether `count` is even *without* side effects. 
-In using the `createEffect` function, the side effects are appropriately handled. 
+In this updated approach, the `isEven` memo will compute based on whether `count` is even *without* side effects.
+However, to minimize unnecessary updates that occur as a result of a change, it is better practice to convert any unnecessary effects into memos.
+This can help to minimize unnecessary updates in other parts of the application.
+And, generally speaing, opting for memos is the recommended approach over the use of effects:
+
+```jsx
+const message = createMemo(() => {
+  if (count() > 10) {
+    return "Count is too high!";
+  } else {
+    return "";
+  }
+});
+
+```
 
-The clear separation of concerns - keeping the computational logic in memos and the side effects in effects - keeps an application's reactivity straightforward and manageable, through reducing the risk of any unintended behaviors.
\ No newline at end of file
+**Note**: In situations where side effects, such as DOM updates, are important, using the `createEffect` function should be used over a memo.
```

Comment:
I wonder about a wording such as this:

> Effects still have their place, e.g. to trigger external side effects such as DOM updates.

I kind of liked your "clear separation sentence", maybe at the very end? But up to you.